ODIN_II/regression_test/benchmark/task/keywords/always/synthesis_result.json

```diff
@@ -0,0 +1,522 @@
+{
+    "always/always_asterisk_event/k6_frac_N10_frac_chain_mem32K_40nm": {
+        "test_name": "always/always_asterisk_event/k6_frac_N10_frac_chain_mem32K_40nm",
+        "architecture": "k6_frac_N10_frac_chain_mem32K_40nm.xml",
+        "verilog": "always_asterisk_event.v",
+        "exit": 1,
```

Comment:
theres no log? whats the failure? we need to parse the root cause of this error if we don't already

Reply:
memory leak

Reply:
Yeah I pulled your branch and I'm staring at it right now to see how I can parse it in effectively with the regex